src/components/elements/input/TimeDurationInput.tsx

```diff
@@ -0,0 +1,65 @@
+import { Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import { memoize } from 'lodash-es';
+import { useMemo } from 'react';
+import NumberInput from '@/components/elements/input/NumberInput';
+import { TextInputProps } from '@/components/elements/input/TextInput';
+
+type Props = { value?: number; onChange?: (val: number | null) => void } & Omit<
+  TextInputProps,
+  'value' | 'onChange'
+>;
+
+const TimeDurationInput = ({ value, onChange, label, ...props }: Props) => {
+  const [hours, minutes] = useMemo(() => {
+    if (!value) return ['', ''];
+    return [Math.floor(value / 60), value % 60];
+  }, [value]);
+
+  const getIntValue = (v?: string | number) => {
+    if (!v) return 0;
+    if (typeof v === 'string') return parseInt(v);
+    return v;
+  };
+
+  const hoursMinutesToValue = memoize(
+    (hours?: string | number, minutes?: string | number) => {
+      const h = getIntValue(hours);
+      const m = getIntValue(minutes);
+      return h * 60 + m;
+    }
+  );
+
+  return (
+    <>
+      {label}
```

Comment:
So I think we want to wrap all of this in a [FormControl](https://mui.com/material-ui/api/form-control/) -- so you have the input label, the 2 inputs, and the FormHelperText.

Currently if I add `helperText` to the item it is showing up twice, once under each input field.

Reply:
Oh thank you! Good point. I had to add `helperText` in explicitly at the bottom, copying an example from `CheckboxGroupInput` and `RadioGroupInputOption`. But I think it looks ok now.